audit-connector/src/main/java/com/sequenceiq/cloudbreak/audit/converter/AuditEventDataUtilizer.java

```diff
@@ -0,0 +1,12 @@
+package com.sequenceiq.cloudbreak.audit.converter;
+
+import com.cloudera.thunderhead.service.audit.AuditProto;
+import com.sequenceiq.cloudbreak.audit.model.ResultEventData;
+
+public interface AuditEventDataUtilizer<D extends ResultEventData> {
```

Comment:
why "D"?
Naming: ResultEventDataConverter (okay, it is not a traditional converter, but still)

Reply:
ResultEventDataUpdater then?

Reply:
it's not updating the ResultEventData but the builder class of the ResultEventData class, hence this choice of name is as straightforward as possible even if it's a little bit longer than it should be